src/native/aws_signing.c

```diff
@@ -74,10 +74,6 @@ static void s_cleanup_callback_data(struct s_aws_sign_request_callback_data *cal
     }
 
     if (callback_data->native_request) {
-        struct aws_input_stream *input_stream = aws_http_message_get_body_stream(callback_data->native_request);
-        if (input_stream != NULL) {
-            aws_input_stream_destroy(input_stream);
-        }
```

Comment:
get this bugfix into main ASAP

Reply:
it's not a bug yet. We haven't release the ownership from user. https://github.com/awslabs/aws-crt-java/pull/436/files#diff-fd13a85f62a17285ea21b70c9e3c83e367b6cc4034d28274ba2ef4d9ea6f4ed0R390